letsencrypt/models/letsencrypt.py

```diff
@@ -8,26 +9,26 @@
 import subprocess
 import tempfile
 from openerp import _, api, models, exceptions
-from openerp.tools.config import _get_default_datadir
+from openerp.tools import config
 
 
 DEFAULT_KEY_LENGTH = 4096
 _logger = logging.getLogger(__name__)
 
 
+def get_data_dir():
+    return os.path.join(config.options.get('data_dir'), 'letsencrypt')
+
+
+def get_challenge_dir():
+    return os.path.join(get_data_dir(), 'acme-challenge')
+
+
 class Letsencrypt(models.AbstractModel):
     _name = 'letsencrypt'
     _description = 'Abstract model providing functions for letsencrypt'
 
     @api.model
-    def get_data_dir(self):
-        return os.path.join(_get_default_datadir(), 'letsencrypt')
-
-    @api.model
-    def get_challenge_dir(self):
-        return os.path.join(self.get_data_dir(), 'acme-challenge')
```

Comment:
yes, that was my misunderstanding about my own code too. Here's the full story: https://github.com/hbrunn/server-tools/pull/2#discussion-diff-63349648